query-builder/src/test/java/com/datastax/oss/driver/api/querybuilder/schema/CreateTableTest.java

```diff
@@ -174,9 +175,15 @@ public void should_generate_create_table_with_options() {
                 .withMinIndexInterval(1024)
                 .withMaxIndexInterval(4096)
                 .withReadRepairChance(0.55)
-                .withSpeculativeRetry("99percentile"))
+                .withSpeculativeRetry("99percentile")
+                .withExtensions(
+                    ImmutableMap.of(
+                        "key1",
+                        "apache".getBytes(StandardCharsets.UTF_8),
+                        "key2",
+                        "cassandra".getBytes(StandardCharsets.UTF_8))))
```

Comment:
Nit: we preserve order of insertion here (by virtue of creating new object instances as we move through the DSL) and all the other examples here are in alphabetical order.  Far from a big deal (it's bordering on trivial actually) but for sake of consistency it's prolly worth moving it up here.

Reply:
Fixed.